app/metal-controller-manager/config/crd/bases/metal.sidero.dev_servers.yaml

```diff
@@ -71,7 +71,7 @@ spec:
                     description: Source for the password value. Cannot be used if Pass is not empty.
                     properties:
                       secretKeyRef:
-                        description: SecretKeyRef defines a ref to a given key within a secret
+                        description: SecretKeyRef defines a ref to a given key within a secret.
```

Comment:
It was missing at #371, and our CI did not catch it

Reply:
I thought this gets generated from the `server_types.go`, so I think _believe_ it should be changed there

Reply:
That's due to missing `check_dirty` pipeline step